client_test.go

```diff
@@ -113,12 +111,14 @@ func TestUpdateFeatureState(t *testing.T) {
 		Type:        "unicode",
 		StringValue: &newFsValue,
 	}
+
+	environmentID := EnvironmentID
```

Comment:
What's the logic here? Do we need to reassign this variable?

Reply:
Yeah, it's because `Environment` in the struct is defined as a pointer, but we can't take the address of a constant(EnvironmentID), that's why we need to create a variable and use its address, a bit mental yes ;)